ui/package.json

```diff
@@ -26,6 +26,7 @@
 	},
 	"type": "module",
 	"dependencies": {
-		"monaco-editor": "^0.34.1"
+		"monaco-editor": "^0.34.1",
+		"svelte-pdf": "^1.0.16"
```

Comment:
We don't need this, because we are using the iframe?

Reply:
solved here: https://github.com/DlieBG/gum/pull/47/commits/7628aa5c2e3dd08dff9e347f17a4f81d793ce193